x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/es_query_expression.tsx

```diff
@@ -244,7 +244,7 @@ export const EsQueryExpression: React.FC<
           </EuiLink>
         }
       >
-        <CodeEditor
+        <CodeEditor // TODO unable to find context. Need CodeOwners to help
```

Comment:
nit: I think we can remove this.

Reply:
Nice catch! Removed here - https://github.com/elastic/kibana/pull/159638/commits/bdbe9a83e97125ba271e026572a0c5818f56dcc2